<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThumbnailerDialog</class>
 <widget class="QDialog" name="ThumbnailerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thumbnailer Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="2">
    <layout class="QGridLayout" name="topright_gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="display_label_3">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>行：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="column_spinBox"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="display_label_4">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>列：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="row_spinBox"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="thumbs_cnt_label">
       <property name="text">
        <string>0 / 9</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QPushButton" name="clear_thumbs_pushButton">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="left_gridLayout">
     <item row="2" column="6">
      <widget class="QComboBox" name="play_spd_comboBox">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;播放速度&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>0.25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4.0</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QPushButton" name="playpause_pushButton">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="stop_pushButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="select_pushButton">
       <property name="text">
        <string>Select Video</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="time_label">
       <property name="text">
        <string>00:00:00/00:00:00</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="7">
      <widget class="QSlider" name="progress_horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="snap_pushButton">
       <property name="text">
        <string>Snap</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="8">
      <widget class="QWidget" name="video_widget" native="true">
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QComboBox" name="volume_comboBox">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;音量&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>0%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>80%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100%</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="Line" name="separator_line">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(134, 134, 134);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="right_gridLayout">
     <item row="2" column="0" colspan="4">
      <widget class="QProgressBar" name="get_thumbs_progressBar">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QListView" name="thumbs_listView">
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="no_watermark_checkBox">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;移除缩略图左上角的时间水印&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>D-M</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="slow_algo_checkBox">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;使用另一种较慢的算法生成缩略图, 兼容性更佳&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Slow</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QPushButton" name="get_thumbs_pushButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="topleft_gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="thumbs_dir_lineEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="current_video_lineEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>No Video Selected.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="display_label_1">
       <property name="text">
        <string>当前视频位置:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="display_label_2">
       <property name="text">
        <string>缩略图生成目录:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="change_dir_pushButton">
       <property name="text">
        <string>更改</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
